models/reviewsModel.js

```diff
@@ -20,15 +20,15 @@ const fetchReviews = () => {
 };
 
 const fetchReviewsById = (reviewId) => {
-  const changeToNumber = +reviewId;
-  if (isNaN(changeToNumber)) {
+  const changedToNumber = +reviewId;
+  if (isNaN(changedToNumber)) {
```

Comment:
We should use SQL errors for this as it will give us an error if review_id does not match the data type expected in the collumn, which we specifiy inside the seed file.